tests/unit/test_command_containers.py

```diff
@@ -33,7 +38,7 @@ def test_render_isolated(runner):
         with open('Dockerfile.j2', 'w') as wh_:
```

Comment:
This one is testing that if we render a Dockerfile.j2 in an environment without git initialized that it still works (specifically that it doesn't try and add the extra git commit environment variable.)